asv/commands/continuous.py

```diff
@@ -123,3 +103,40 @@ def results_iter(commit_hash):
             color_print("BENCHMARKS NOT SIGNIFICANTLY CHANGED.", 'green')
 
         return worsened
+
+
+def _results_iter(commit_hash, run_objs, result):
+    """
+    Iterate over results for the given commit_hash.
+
+    Parameters
+    ----------
+    commit_hash : str
+    run_objs : dict
+    result : results.Results
+
+    Yields
+    ------
+    name : str
+    params : list
+    value : object
+    stats : list
+    version : str
+    """
+    for env in run_objs['environments']:
+        machine_name = run_objs['machine_params']['machine']
+        filename = results.get_filename(machine_name, commit_hash, env.name)
+        filename = os.path.join(conf.results_dir, filename)
```

Comment:
`conf` is undefined.

I'd prefer to keep this as an inline function in the Continuous method, however, it's short enough and the logic belongs there.

Reply:
OK, I'll revert this part of the PR.